docs/index.html

```diff
@@ -748,7 +616,11 @@ <h3>What is the purpose?</h3>
 <h3>How do we maintain it?</h3>
 
 <p>
-The Tech Radar is maintained by the <em>Runtastic Architecture Guild</em>. The Tech Radar depends on active participation and input from all engineering teams.
+The Tech Radar is maintained by the <em>Runtastic Architecture Guild</em>. 
+The Tech Radar depends on active participation and input from our entire tech organization. 
+The architecture team drives revisiting the Tech Radar twice a year, actively asking for feedback around potential updates. 
+Besides that please use the #ask-the-architects slack channel for discussions and information about new technology adoption.
+Please also refer to <a href="https://runtastic.stoplight.io/docs/tech-radar-insights/docs/readme.md">Stoplight</a> for detailed information on some of the technologies.
```

Comment:
What's the actual name of this documentation? I wouldn't really say "refer to stoplight" but rather "refer to the tech radar insights documentation" or something like that.

stoplight is a tool not a specific documentation